README.md

```diff
@@ -1,2 +1,151 @@
-# Enumerable
-n this project, a program was written that takes advantage of the built in ruby enumerable module
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://raw.githubusercontent.com/microverseinc/readme-template/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Enumerable](#-enumerable)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Author ](#-author-)
+  - [Author](#author)
+  - [Author](#author-1)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments \& Attribution ](#-acknowledgments--attribution-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Enumerable<a name="about-project"></a>
+
+**Enumerable**In this project, a program was written that takes advantage of the built in ruby enumerable module.
```

Comment:
- [ ] Kindly make this section more descriptive, for example, the heding could read `Create Your Own Enumerable`.
- [ ] the actual description should explain what the project does rather than what language it was built in